### DISCUSSION TITLE
Custom status

### DISCUSSION THREAD

**Discussion by lodipo308** [Rapptz/discord.py#8021] [Q&A] [answered]
How can I use discord py to set custom status on my Discord account? I know that custom statuses don't work for bots but I want to change it for user account with updated stats about something.

**Comment by LostLuma** [comment, marked as answer]
You can not set a custom status on a user account in any TOS-friendly way sadly. The best alternative is to create a program to set a rich presence status instead, however this is not a part of discord.py, so you will need to find another library to do so.

**Reply by lodipo308** [reply]
I just want to try it, I dont care about Discord TOS

**Reply by lodipo308** [reply]
```
import discord
import os
from discord.ext import commands

client = commands.Bot(command_prefix=':', self_bot=True, help_command=None)



@client.event
async def on_ready():
    print('Logged in')
    await client.change_presence(activity=discord.Activity(
        type=discord.ActivityType.watching, name="my text"),
                                 status=discord.Status.dnd)


client.run(os.getenv("TOKEN"), bot=False)
```
This is the closest I get to, but its watching activity, not custom user status
